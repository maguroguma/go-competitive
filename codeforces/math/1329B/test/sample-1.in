10
1 1000000000
2 999999999
3 99999998
4 9999997
5 999996
6 99995
7 9994
8 993
9 92
10 1
