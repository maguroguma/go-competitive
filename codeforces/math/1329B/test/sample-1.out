1
3
5
11
17
23
29
59
89
0
